### PR TITLE
Add feedback form

### PR DESCRIPTION
**Pull request by bangunbagustapa**
### Changes

- Add feedback form in hero section and footer section

### Tasks

- [S2A4 - Feedback/Umpan balik](https://airtable.com/app7SdZInMN1pG6ZL/tblLy5A3qYF19fj1u/viwDUthwzD6mc5jka/rechecLENMZZS0K4s?blocks=hide)
- [S8A3 - Umpan Balik](https://airtable.com/app7SdZInMN1pG6ZL/tblLy5A3qYF19fj1u/viwDUthwzD6mc5jka/recIQOekmyA4Mfiaj?blocks=hide)

### Preview

feedback

### CODE REVIEW

**Review by Ibwedagama — APPROVED** [review]

**Review by maulanayuseph — APPROVED** [review]

**Review by maruf12 — APPROVED** [review]

File: components/Feedback/index.vue [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+        }
+        if (cookie.indexOf(name) === 0) {
+          return cookie.substring(name.length, cookie.length)
+        }
+      }
+
+      return ''
+    },
+    setCookie (key, value) {
+      document.cookie = `${key}=${value};`
```
Comment (adzharamrullah) [inline components/Feedback/index.vue:203]: I wonder, why you store data to `cookie` why not using `localStorage` ? what your concern to do it ?

File: components/Feedback/index.vue [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+  data () {
+    return {
+      form: {
+        rating: null,
+        compliments: '',
+        criticism: '',
+        suggestions: ''
+      },
+      isLoading: false,
+      ratingOptions: [
```
Comment (adzharamrullah) [inline components/Feedback/index.vue:135]: do you have any errors define options to radio button group with type of property value is number ?. i found bug in radio button group component which is component not passed value except string, that should be passed other  data type like number, string, object, boolean

File: components/Feedback/index.vue (hunk 1 above)
Comment (bangunbagustapa) [reply components/Feedback/index.vue]: In this case, both will be working fine. I [moved](https://github.com/jabardigitalservice/portal-jabar-frontend/pull/29/commits/c4f97603b8ad6f5a56843c108772760b6860a2f9) to `localstorage` due to simplicity. Thank you @adzharamrullah.

File: components/Feedback/index.vue (hunk 2 above)
Comment (bangunbagustapa) [reply components/Feedback/index.vue:135]: It's working fine. Maybe if we use `v-model`, the type must be `string`.

**Review by adzharamrullah — APPROVED** [review]

**Pull request merged**